cmstestsuite/testrunner.py

```diff
@@ -278,20 +278,29 @@ def _all_user_tests(self):
                 for lang in test.languages:
                     yield (test, lang)
 
-    def submit_tests(self):
+    def submit_tests(self, concurrent_submit_and_eval=True):
         """Create the tasks, and submit for all languages in all tests.
 
+        concurrent_submit_and_eval (boolean): if False, stop ES while
+            we are submitting, so that we have a clearer view of the time
+            each step takes.
+
         """
+        def start_eval_services():
```

Comment:
There's a discrepancy between what this function does and what the docstring says it does.

Reply:
Done